src/clients/mocks/MockConfigStoreClient.ts

```diff
@@ -23,11 +23,10 @@ export class MockConfigStoreClient extends AcrossConfigStoreClient {
     configStore: Contract,
     eventSearchConfig: MakeOptional<EventSearchConfig, "toBlock"> = { fromBlock: 0, maxBlockLookBack: 0 },
     configStoreVersion: number,
-    enabledChainIds: number[],
     chainId = 1,
     mockUpdate = false
   ) {
-    super(logger, configStore, eventSearchConfig, configStoreVersion, enabledChainIds);
+    super(logger, configStore, eventSearchConfig, configStoreVersion);
```

Comment:
I think we're going to actually want to mock the `getEnabledChains()` function and have it return `enabledChainIdsOverride` if there is no GlobalConfig update. does that make sense?

Reply:
Follwed up here: https://github.com/across-protocol/sdk-v2/pull/346/commits/9533b39849528443840d6329d3249144017bb2db